test/unit/app/models/consultation_response_form_data_test.rb

```diff
@@ -15,4 +15,18 @@ class ConsultationResponseFormDataTest < ActiveSupport::TestCase
 
     assert_equal consultation_response_form_data.auth_bypass_ids, [auth_bypass_id]
   end
+
+  test "asset_uploaded?: should return true when there is original" do
```

Comment:
I believe this is outdated, seems to have been renamed to `all_asset_variants_uploaded?` 
Also typo in `bo` -> `no` 😅
And we can use the hash thing`#all_asset_variants_uploaded?` in the test name.